tests/test_refspec/test5.yml

```diff
@@ -0,0 +1,9 @@
+header:
+  version: 8
+
+repos:
+  this:
+
+  kas:
+    url: https://github.com/siemens/kas.git
+    refspec: master@907816a5c4094b59a36aec12226e71c461c05b77
```

Comment:
Patch ordering: test should follow the new feature, not the other way around.

Reply:
I agree. I will reverse the order.